src/uipath/_cli/_utils/_parse_ast.py

```diff
@@ -481,7 +482,7 @@ def convert_to_bindings_format(sdk_usage_data) -> Bindings:
             folder_path = folder_path.replace("EXPR$", "") if folder_path else None
             key = name
             if folder_path:
-                key = f"{folder_path}.{name}"
+                key = f"{name}.{folder_path}"
```

Comment:
This seems like a big change -- could you share more details on the run configuration contract?

Regardless, can we move this change to a separate PR for separation of concerns?